src/main/java/com/minecolonies/coremod/items/ItemScepterGuard.java

```diff
@@ -118,10 +118,10 @@ private static EnumActionResult handleItemUsage(final World worldIn, final Block
             name = " " + citizen.getName();
         }
 
-        if (task.equals(Task.GUARD))
+        if (task.equals(GuardTask.GUARD))
         {
             LanguageHandler.sendPlayerMessage(playerIn, "com.minecolonies.coremod.job.guard.toolClickGuard", pos, name);
-            tower.setGuardTarget(pos);
+            tower.setGuardPos(pos);
```

Comment:
it's more of a target no?

Reply:
it's either or. both names fit. The GuardTower variable that that method assigns is `guardPos` soooo... 
I personally think guardPos is more descriptive than a guardTarget (Could be taken as an entity or anything)